lib/msal-browser/test/interaction_client/NativeInteractionClient.spec.ts

```diff
@@ -1054,6 +1056,25 @@ describe("NativeInteractionClient Tests", () => {
                 failedRequests: [],
             });
         });
+
+        it("should not include onRedirectNavigate call back function in request", (done) => {
+            jest.spyOn(
+                NativeInteractionClient.prototype,
+                // @ts-ignore
+                "initializeNativeRequest"
+                // @ts-ignore
+            ).mockImplementation((request: PopupRequest | SsoSilentRequest) => {
+                // @ts-ignore
+                expect(request.onRedirectNavigate).toBeUndefined();
+                done();
+            });
+            nativeInteractionClient.acquireTokenRedirect(
+                {
+                    scopes: ["User.Read"],
```

Comment:
You're missing `onRedirectNavigate` here. This test will never fail as written :smile:

Reply:
Oh no! Fixed